docker-storage-setup.sh

```diff
@@ -848,6 +848,17 @@ reset_storage() {
     rm -f $DOCKER_STORAGE
 }
 
+# Config dumping
+
+dump_config() {
+    echo STORAGE_DRIVER=$STORAGE_DRIVER
+    echo VG=$VG
+    if [ "$STORAGE_DRIVER" == "devicemapper" ]; then
+        echo LV=$POOL_LV_NAME
+    fi
+    echo DEVS=$DEVS
+}
+
```

Comment:
> What's future can easily be parsed from /etc/sysconfig/docker-storage-setup anyway.

"easily" is relative. /e/s/d-s-s is a shell script, and the only way to parse it correctly is to execute it.  You can think of `--dump-config` as a utility that parses /e/s/d-s-s for you.

Also, default values are not "easily" computed, such as digging out the volume group behind "/" as the default value for $VG.  `--dump-config` will also compute these default values correctly for you.

Thus, even if a real `--status` is more important than a mere `--dump-config`, `--dump-config` is still useful since it's the best way to parse /e/s/d-s-s correctly if that's what you want.  (We could make it output everything and carefully document the format, though.  Maybe even make it optionally output JSON because inventing our own key/value syntax is a sin. :-)